apps/src/lab2/views/components/Instructions/ResourcePanel/useOnboardingTour.tsx

```diff
@@ -0,0 +1,56 @@
+import {Steps} from 'intro.js-react';
+import React, {useMemo} from 'react';
+
+import {PYTHONLAB_RESOURCE_PANEL_ONBOARDING_TOUR_SEEN} from '@cdo/apps/pythonlab/constants';
```

Comment:
Nit: I think this constant should go under `lab2` since it's only used within the lab2 folder. Or we should move these hooks into python lab and pass in generic `steps` to the resource panel.